doc-content/enterprise-only/installation/clustering-smart-router-install-proc.adoc

```diff
@@ -7,6 +7,15 @@ This section describes how to install Smart Router and configure it for a Red Ha
 
 .Prerequisites
 * {KIE_SERVER} is installed on each node of a {EAP} {EAP_VERSION} cluster.
++
+[NOTE]
+====
+Smart Router does not require {KIE_SERVER} to be configured as unmanaged.
+
+An unmanaged {KIE_SERVER} does not connect to the controller.
+
+If you connect an unmanaged {KIE_SERVER} to a Smart Router and register the Smart Router with the controller, {CENTRAL} can contact the unmanaged {KIE_SERVER} by using Smart Router.
```

Comment:
Suggested rewrite:

For instance, If you connect an unmanaged {KIE_SERVER} to a Smart Router and register the Smart Router with the controller, then the {CENTRAL} contacts the unmanaged {KIE_SERVER} by using Smart Router. //Adding an example form.

Reply:
I like the suggestion, we can remove the articles for {CENTRAL} and Smart Router and capitalized "if". Is this ok?

"For instance, if you connect an unmanaged {KIE_SERVER} to Smart Router and register the Smart Router with the controller, then {CENTRAL} contacts the unmanaged {KIE_SERVER} by using Smart Router."

Reply:
yes, its good, thanks